### DISCUSSION TITLE
Store debug info (motion boxes, etc) for recorded clips?

### DISCUSSION THREAD

**Discussion by ireuben** [blakeblackshear/frigate#2598] [Ideas]
Would it be useful to be able to view debug info for any previously recorded clip, so you can play a clip and see what exactly triggered the motion, etc in that clip? Seems like that could potentially be stored in the db, and drawn while playing back the clip.

**Comment by blakeblackshear** [comment]
This will be done by rerunning recorded footage through processing as described here: https://github.com/blakeblackshear/frigate/issues/1914
